stacker/providers/session_cache.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import logging
+import botocore
+
+
+def get_session(region):
+    session = botocore.session.get_session()
+    session.set_config_variable('region',  region)
+    c = session.get_component('credential_provider')
+    provider = c.get_provider('assume-role')
```

Comment:
There will be no issues, assume-role is just where botocore stores all the role info.